back/engines/free/seo/app/controllers/seo/application_controller.rb

```diff
@@ -16,7 +16,7 @@ def sitemap
         :'admin_publications.publication_type', :'admin_publications.publication_id'
       ).includes(:admin_publication).where(visible_to: 'public', admin_publications: { publication_status: statuses })
       @initiatives = Initiative.select(:slug, :updated_at, :publication_status).where(publication_status: statuses)
-      @pages       = Page.select(:slug, :updated_at, :publication_status).where(publication_status: statuses)
+      @pages       = StaticPage.select(:slug, :updated_at, :publication_status).where(publication_status: statuses)
```

Comment:
`static_pages` doesn't have `publication_status` column anymore, does it?

And this file is covered with tests according to my generated report. I cannot understand why it didn't fail on CI. Am I missing something?

Reply:
Very well spotted! Indeed the tests didn't detect this. Fixing it.

Reply:
I figured it out :) It's because the pages are not included in the sitemap if the pages feature is not enabled. Since I strongly suspect we still want to list e.g. the privacy policy even when this feature is disabled, I'm changing the code to also list pages when disabled and am improving the spec to also create a page and initiative to include in the sitemap.

Reply:
:+1: 

Ah, this line didn't raise an error in tests because the association is not loaded.